src/core/lifetime.d

```diff
@@ -2124,7 +2124,9 @@ private void moveEmplaceImpl(T)(scope ref T target, return scope ref T source)
                 () @trusted { memset(&source, 0, sz); }();
             else
             {
-                auto init = typeid(T).initializer();
+                import core.internal.lifetime : emplaceInitializer;
+                ubyte[T.sizeof] init;
```

Comment:
```suggestion
                ubyte[T.sizeof] init = void;
```

Reply:
There's no point in initializing it before emplacing something in it.

Reply:
Thanks, I've overlooked that.
I've squashed the commits.